static/app/components/loadingTriangle.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+
+import sentryLoader from 'sentry-images/sentry-loader.svg';
+
+import space from 'sentry/styles/space';
+
+type Props = {
+  children?: React.ReactNode;
```

Comment:
```suggestion
  children: React.ReactNode;
```

in this case, `children` does not need to be optional as the ReactNode can be of type `undefined` or `null`

`  type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;`

Reply:
Ah. Let's leave it as optional since IMO it makes it more explicit when looking at the type. What do you think?